### DISCUSSION TITLE
[Feature request] Address bar from before 1909

### DISCUSSION THREAD

**Discussion by aubymori** [valinet/ExplorerPatcher#947] [Ideas]
Before Windows 10 1909, the address bar was smaller, and the search was different. You can revert it, and in 1909 it reverts the size, but in anything newer the size stays the same. Maybe you could add an option for that smaller address bar? A lot of people including me like it better than the newer one. I've experimented a bit with changing the size in Window Detective and it seems pretty trivial.

**Comment by valinet** [comment]
Personally, idk how to change its size, I mean, I have ideas ofc, but no plan, have never studied it in-depth. If you say it's doable, contribute some patch where you address this. Changing the size manually doesn't cut it, I think, as there are a lot of cases that then had to be treated. Instead, what would work better is to determine what in the code they changed that produces the bigger size, and alter that back. There is something in there that dictates the size, as the "Address" deskband (taskbar toolbar) uses the same component and it too is bigger.

**Reply by aubymori** [reply]
I don't know C, so sadly I can't contribute.

**Reply by valinet** [reply]
But do you know how to make it work reliably, that was my question? The language doesn’t matter that much, after all, if it’s Turing complete, it’s fine.

**Comment by XuanChen1234** [comment]
@Vipin2003 @Amrsatrio

**Reply by valinet** [reply]
@XuanChen1234 You have been blocked on my account for repeatedly spamming my mail inbox with unnecessary tagging.

**Reply by vippium** [reply]
Finally, vali looked at him...hasshhhhh

**Comment by PandaFiredoge** [comment]


**Reply by MrRedstonia** [reply]
really..?

**Reply by PandaFiredoge** [reply]
Sorry, this is not a reply to this discussion
